src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -79,6 +79,7 @@ memgraphCypherKeyword : cypherKeyword
                       | SESSION
                       | SETTING
                       | SETTINGS
+                      | SHOW_CONFIG
```

Comment:
I think this is wrong. In the query you use them as separate words (`SHOW CONFIG`), and not as one (like `BATCH_LIMIT`), thus I think you need `SHOW` and `CONFIG` as separate words. I think both words are there (`SHOW` is in `Cypher.g4` in the `cypherKeyword` rule, `CONFIG` is here). So I think you don't have to modify the keywords unless you want to introduce another word.